google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/DefaultCredentialProvider.java

```diff
@@ -259,38 +258,18 @@ private GoogleCredential getCredentialUsingWellKnownFile(
   }
 
   private boolean runningOnAppEngine() {
-    Class<?> systemPropertyClass = null;
-    try {
-      systemPropertyClass = forName("com.google.appengine.api.utils.SystemProperty");
-    } catch (ClassNotFoundException expected) {
-      // SystemProperty will always be present on App Engine.
-      return false;
+    Map<String, String> env = System.getenv();
+    if (env.containsKey("GAE_ENV") && env.get("GAE_VM").equals("standard")) {
```

Comment:
I don't think GAE_VM ever equals "standard"

Reply:
I think you meant to check `GAE_ENV`, not `GAE_VM`

Reply:
Indeed, copy and paste error… Thanks